pkg/cmd/install/hubaddon/scenario/addon/appmgr/service_metrics.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright Contributors to the Open Cluster Management project
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: subscription-operator-metrics
+  name: subscription-operator-metrics
+  namespace: open-cluster-management
+spec:
+  ports:
+    - name: metrics
+      port: 8381
+      protocol: TCP
+      targetPort: 8381
+  selector:
+    name: multicluster-operators-subscription
```

Comment:
Selector is not correct. Should be multicluster-operators-hub-subscription
FYI @xiangjingli 
```➜  bin oc get pods -n open-cluster-management | grep multicluster-operators-hub-subscription
multicluster-operators-hub-subscription-f78bcd465-rwd4w          1/1     Running   0          4h6m
➜  bin oc get pods -n open-cluster-management multicluster-operators-hub-subscription-f78bcd465-rwd4w --show-labels
NAME                                                      READY   STATUS    RESTARTS   AGE    LABELS
multicluster-operators-hub-subscription-f78bcd465-rwd4w   1/1     Running   0          4h6m   app=multicluster-operators-hub-subscription,pod-template-hash=f78bcd465
➜  bin
```

Reply:
actually... this doesn't work well for installations performed with `clusteradm` which uses a [different selector labels](https://github.com/open-cluster-management-io/clusteradm/blob/main/pkg/cmd/install/hubaddon/scenario/addon/appmgr/deployment_subscription.yaml#L17)

```shell
$ oc get pods -n open-cluster-management multicluster-operators-subscription-5c9b4cbdc8-p4rsk --show-labels

NAME                                                   READY   STATUS    RESTARTS      AGE   LABELS
multicluster-operators-subscription-5c9b4cbdc8-p4rsk   1/1     Running   1 (27m ago)   38m   name=multicluster-operators-subscription,pod-template-hash=5c9b4cbdc8
```

Reply:
I also just bumped into this [standalone deployment resource file](https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/main/deploy/standalone/operator.yaml#L7) which I assume is used for manual installations of the standalone environment, it also uses the `name: multicluster-operators-subscription` label.

Reply:
FYI @xiangjingli Is the upstream name expected to be different from midstream?

Reply:
@TomerFi @chenz4027  I would suggest to apply different labels to hub subscription pod and standalone subscription pod in the upstream, as what we do in the midstream.

- For hub subscription pod, apply the label to here  
[different selector labels](https://github.com/open-cluster-management-io/clusteradm/blob/main/pkg/cmd/install/hubaddon/scenario/addon/appmgr/deployment_subscription.yaml#L17)
```
app: multicluster-operators-hub-subscription
```

- For standalone subscription pod, apply the label
[standalone deployment resource file](https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/main/deploy/standalone/operator.yaml#L7)
```
app: multicluster-operators-standalone-subscription
```

Reply:
@xiangjingli 
I wonder... should we perhaps compare other resources/properties to the midstream...
i.e. _multicluster-operators-hub-subscription_ instead of _multicluster-operators-subscription_ for the pod name.

there are also a couple diffs in the appsubsummary deployment, as well as the service account name.
so this might be a breaking change for clusteradm users, but maybe it's worth it if it'll straighten the line with the midstream.